### ISSUE TITLE
PieChart labels on wrong positions

### ISSUE THREAD

**Issue by ruzovvo** [beto-rodriguez/Live-Charts#350]
Hi Beto,
I've already reported an issue with PieChart labels is "Realtime changes on PieChart #47 "

Now I upgraded to 0.7.17.0 and faced the similar issue again
Please check the screenshot:
![image](https://cloud.githubusercontent.com/assets/4226718/19213430/25289a32-8d75-11e6-9290-bb48a7c0fddc.png)
#### How to reproduce?

My ViewModel:

``` c#
using Test.Models;
using LiveCharts;
using log4net;
using LiveCharts.Defaults;
using LiveCharts.Wpf;

namespace Test.ViewModels.Indexes
{
    public class SpectrumViewModel : DockWindowViewModel
    {
        public HRIndexViewer IndexViewer { get; set; }
        public User User { get; set; }
        public static readonly ILog Logger = LogManager.GetLogger(typeof(SpectrumViewModel));

        private SeriesCollection _series;
        public SeriesCollection Series
        {
            get
            {
                if (_series != null) return _series;
                //var config = new SeriesConfiguration<StoreViewModel>().Y(y => y.Value);
                _series = new SeriesCollection();
                return _series;
            }
        }

        #region Ctor
        public SpectrumViewModel( HRIndexViewer indexViewer, User user ) : base("Spectrum")
        {
            Title = "Spectrum";
            IndexViewer = indexViewer;
            User = user;
            Init();
        }

        public SpectrumViewModel() : base("Spectrum")
        {
        }

        #endregion

        private void Init()
        {
            Series.Add(new PieSeries()
            {
                Title = "ULF, %",
                DataLabels = true,
                Values = new ChartValues<ObservableValue>()
                {
                    IndexViewer.UlfPercents
                },
            });
            Series.Add(new PieSeries()
            {
                Title = "VLF, %",
                DataLabels = true,
                Values = new ChartValues<ObservableValue>()
                {
                    IndexViewer.VlfPercents
                },
            });
            Series.Add(new PieSeries()
            {
                Title = "LF, %",
                DataLabels = true,
                Values = new ChartValues<ObservableValue>()
                {
                    IndexViewer.LfPercents
                },
            });
            Series.Add(new PieSeries()
            {
                Title = "HF, %",
                DataLabels = true,
                Values = new ChartValues<ObservableValue>()
                {
                    IndexViewer.HfPercents
                },
            });
        }
    }
}
```

My xaml snippet:

``` xml
<wpf:PieChart LegendLocation="Right" Grid.Row="0" Grid.Column="0" 
                             DisableAnimations="True" 
                             Series="{Binding Series}" InnerRadius="30">
            <wpf:PieChart.ChartLegend>
                <wpf:DefaultLegend BulletSize="20"></wpf:DefaultLegend>
            </wpf:PieChart.ChartLegend>
            <wpf:PieChart.DataTooltip>
                <wpf:DefaultTooltip BulletSize="20"></wpf:DefaultTooltip>
            </wpf:PieChart.DataTooltip>
            <wpf:PieChart.AxisY>
                <wpf:Axis Title="Spectrum, %" />
            </wpf:PieChart.AxisY>
            <wpf:PieChart.AxisX>
                <wpf:Axis Title="Frequences" />
            </wpf:PieChart.AxisX>
        </wpf:PieChart>
```
#### Extra notes

Maybe I use smth in wrong way as you've already fixed it. Can you please advise?
#### I am using

<table>
<tr>
    <td>Live-Charts version</td>
    <td>0.7.17.0</td>
</tr>
<tr>
    <td>.Net Version</td>
    <td>4.5</td>
</tr>
<tr>
    <td>Windows</td>
    <td>10</td>
</tr>
</table>

**Comment by rusanov-vladimir** [comment]
Same problem here:
![labels](https://cloud.githubusercontent.com/assets/3341799/19852588/edd7baa4-9f6c-11e6-968a-5a9e76aaac02.png)

**Comment by beto-rodriguez** [comment]
@rusanov-vladimir Nice catch! that helps alot!

**Comment by beto-rodriguez** [comment]
This is already fixed and will be included in since 0.8

Thanks for the feedback!
